include/experimental/__p0009_bits/layout_left.hpp

```diff
@@ -108,6 +109,31 @@ class layout_left::mapping {
         */
     }
 
+#if MDSPAN_HAS_CXX_17
+    MDSPAN_TEMPLATE_REQUIRES(
+      class _Mapping,
+      /* requires */ (
+        detail::__is_layout_left_padded_mapping<_Mapping>::value
+        && is_constructible_v<extents_type, typename _Mapping::extents_type>
+      )
+    )
+    MDSPAN_CONDITIONAL_EXPLICIT((!is_convertible_v<typename _Mapping::extents_type, extents_type>))
+    mapping(const _Mapping& __other) noexcept
+      : __extents(__other.extents())
+    {
+      if constexpr ((extents_type::rank() > 1)
+                    && (extents_type::static_extent(0) != dynamic_extent)
+                    && (_Mapping::extents_type::static_extent(0) != dynamic_extent)
+                    && (detail::__padded_layout_padding_stride<typename _Mapping::layout_type>::value != dynamic_extent)) {
+        if constexpr (extents_type::static_extent(0) == 0) {
+          static_assert(detail::__padded_layout_padding_stride<typename _Mapping::layout_type>::value == 0);
+        } else {
+          static_assert(detail::__padded_layout_padding_stride<typename _Mapping::layout_type>::value % extents_type::static_extent(0) == 0);
```

Comment:
Doesn't this need to be the other way around? 

```
extents_type::static_extent(0) % detail::__padded_layout_padding_stride<typename _Mapping::layout_type>::value == 0
```

Reply:
woops, yeah, fixed